packages/tools/src/tools/annotation/ArrowAnnotateTool.ts

```diff
@@ -409,13 +403,16 @@ class ArrowAnnotateTool extends AnnotationTool {
     triggerAnnotationRenderForViewportIds(renderingEngine, viewportIdsToRender);
   };
 
-  doubleClickCallback = (evt: EventTypes.MouseUpEventType) => {
+  touchTapCallback = (evt: EventTypes.TouchTapEventType) => {
+    if (evt.detail.taps == 2) {
+      this.doubleClickCallback(evt as unknown as EventTypes.MouseUpEventType);
```

Comment:
this can pass evt without `as unknown as EventTypes...` with the `InteractionEventType` change we made

Reply:
Actually, I think we need to use the specific TouchTapEventDetail here because we don't have the "taps" parameter from InteractionEventType. So "TouchTapEventDetail" is inherited from the interaction.

```
type TouchTapEventDetail = NormalizedInteractionEventDetail &
  TouchCustomEventDetail & {
    currentPointsList: ITouchPoints[];
    currentPoints: ITouchPoints;
    taps: number; <----
  };
```

Reply:
As we discussed yesterday, this is a special case where we need to use the taps parameter and we don't have it inherited from the InteractionType. So, make sense to use the TouchTapEventDetail at this specific point.